src/Common/ZooKeeper/ZooKeeperCommon.cpp

```diff
@@ -933,40 +1123,6 @@ std::shared_ptr<ZooKeeperRequest> ZooKeeperRequest::read(ReadBuffer & in)
     return request;
 }
 
-ZooKeeperRequest::~ZooKeeperRequest()
-{
-    if (!request_created_time_ns)
-        return;
-    UInt64 elapsed_ns = clock_gettime_ns() - request_created_time_ns;
-    constexpr UInt64 max_request_time_ns = 1000000000ULL; /// 1 sec
-    if (max_request_time_ns < elapsed_ns)
-    {
-        LOG_TEST(getLogger(__PRETTY_FUNCTION__), "Processing of request xid={} took {} ms", xid, elapsed_ns / 1000000UL);
```

Comment:
Why do we delete these test logs?

Reply:
because they are useless and have a huge impact on performance
when you have a Multi request with a lot of requests and a destructor is called we call this function for each subrequest
If you have a lot of Multi request this can impact performance heavily
Also Keeper will now print similar logs about slow request processing in much more details